lrspline/__init__.py

```diff
@@ -461,18 +479,32 @@ def derivative(self, u, v, d=(1,1), iel=-1):
             return np.array(retval).reshape(u.shape)
         return self.w.point(u, v, nderivs, iEl=iel)[index]
 
+    def getBezierExtraction(self, iEl):
```

Comment:
I'm trying to control names better in the Pythonic interface. I'd prefer `bezier_extraction`.